tests/crud/test_crud_endpoints.py

```diff
@@ -282,6 +279,24 @@ def test_patch_fails(self):
         response = client.patch(f"/{movie.id}/", json={"foo": "bar"})
         self.assertEqual(response.status_code, 400)
 
+    def test_patch_validation_error(self):
+        """
+        Check if Pydantic validation error works.
+        """
+        client = TestClient(PiccoloCRUD(table=Movie, read_only=False))
+
+        movie = Movie(name="Star Wars", rating=93)
+        movie.save().run_sync()
+
+        response = client.patch(
+            f"/{movie.id}/",
+            json={"name": 95, "rating": "95"},
+        )
+        self.assertIn("validation error", str(response.content))
```

Comment:
Thanks for adding this.

In Pydantic v2, the validation error we get back is pretty verbose / technical:

```
b'1 validation error for MovieOptional\nname\n  Input should be a valid string [type=string_type, input_value=95, input_type=int]\n    For further information visit https://errors.pydantic.dev/2.3/v/string_type'
```

I guess it's OK.